### PR TITLE
Add display inline-grid and ::placeholder pseudo-element

### PR DESCRIPTION
**Pull request by LooserName404**
This PR is to add my suggestions from the issues #14 and #15  to the library.

Feel free to point out any errors or even reject the suggestions.

### CODE REVIEW

File: Fss.Core/Types/MasterTypes.fs [hunk 1]
```diff
@@ -1,6 +1,8 @@
 namespace Fss
 namespace Fss.Types
 
+open Fss.Types
```
Comment (Bjorn-Strom) [inline Fss.Core/Types/MasterTypes.fs:4]: This opening statement is not needed I think

File: tests/Tests.fs [hunk 2]
```diff
@@ -72,7 +72,7 @@ module Tests =
                 AnimationTests.tests
                 FontTests.tests
                 TransformTests.tests
-                PsuedoTests.tests
+                PseudoTests.tests
```
Comment (Bjorn-Strom) [inline tests/Tests.fs:75]: Nice! 🤩

**Review by Bjorn-Strom — CHANGES_REQUESTED** [review]
Hey @LooserName404!
This is excellent work. Thank you so much for the contribution.

I only have one tiny comment and that is to remove an import I think is unneeded.
I will definitely merge and release this as soon as that is fixed.

Thanks again, this is great! 👍

File: Fss.Core/Types/MasterTypes.fs (hunk 1 above)
Comment (LooserName404) [reply Fss.Core/Types/MasterTypes.fs]: Sorry, didn't noticed it. Fixed.

**Pull request merged**